src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : Shape
+  color : Color;
+  getArea() : number;
 }
 
-export class Triangle {
+let square : number;
```

Comment:
You can just pass required variable into function and return value you need to have.

Reply:
Ok, thanks